shared/validation/languages/th.js

```diff
@@ -0,0 +1,64 @@
+// We count chars to validate instead of words.
+const MIN_LENGTH = 2;
+const MAX_LENGTH = 150;
```

Comment:
how long does it approximately take to read out a sentence with 150 characters? Of course it will be depending on the actual words etc, but looking for a guessed average here.

Reply:
The number is now a real guess work. Do you have a suggestion for the ideal length/range, in terms of time? 30 seconds? 20 seconds?

Reply:
We're looking for around 7-10 seconds maximum.

Reply:
I have tried reading around 30 sentences from the sentence collector review, 7-10 seconds could be translated to 80-100 characters. I put it 110 chars to allow some more room.